README.md

```diff
@@ -2,12 +2,22 @@
 
 (Not only) coding guidelines and best practices hub.
 
+
+
 ## Azure
 
 - [Naming Resources](azure/naming_resources.md)
 
 
 
+## Glossary
+
+- [Control Plane](glossary/control_plane.md)
+- [Installation](glossary/installation.md)
+- [Tenant Cluster](glossary/tenant_cluster.md)
```

Comment:
I feel like the whole glossary should be rather a single page. You can maybe put it under `General` section. I can see over time becoming really long list of single sentence pages.

Reply:
I found it quite cool to see the separate files and the list in the readme. I don't think this list will become very long. We are not going to have more than 10 GS specific terms to talk about I guess. Or, I hope. :D

Reply:
I'd also be fine with a single page. But no blocker if you'd prefer them to be separate.

Reply:
I'd also prefer single page for easier navigation (or more like lack of unneeded navigation), but no blocker for me as well.

Reply:
When the majority thinks a single page is better, let's do this when the list gets longer, or feel free to provide a PR. Thank you for your feedback.